ceph/rados/bluestoretool_workflows.py

```diff
@@ -33,7 +33,7 @@ class BluestoreToolWorkflows:
     Contains various functions to verify ceph-bluestore-tool commands
     """
 
-    def __init__(self, node: CephAdmin):
+    def __init__(self, node: CephAdmin, nostop=None, nostart=None):
```

Comment:
please add a safety check such that both nostop & nostart vars cannot be sent as true for a single object.

Reply:
Why can't both be set to true? They are mutually exclusive